client/src/components/Header/components/skip-to-content-button.css

```diff
@@ -0,0 +1,10 @@
+.skip-to-content-button {
+  position: absolute;
+  top: var(--header-height);
+}
+.skip-to-content-button.hide-skip-button {
+  left: -1000px;
+}
+.skip-to-content-button.show-skip-button {
+  left: 0px;
+}
```

Comment:
the `index.tsx` page is using `header.css`, so there is no need to create new file

```suggestion
```

if you decide to move it, could you change the styles too?

You can add some basic styles for now, maybe I will discuss with ahmad how to style it, but that is for different PR.

```css
.skip-to-content-button {
  position: absolute;
  top: var(--header-height);
  border-radius: 15px;
  font-weight: 600;
  padding-block: 1em;
  padding-inline: 1.5em;
  left: -1000px;
}

.skip-to-content-button:focus {
  left: 0px;
}
```